src/modules/ROOT/pages/changelog/202209/09.json

```diff
@@ -0,0 +1,120 @@
+      ],
+      "how": [
+        "by adding an expression handler for the function and only allowing it for MySQL databases."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/3ec7d19fae01ddbc4d8effb733a4650dad912cfe"
+      ]
+    },
+    {
+      "what": [
+        "record lists raising Could not load record list: can not change module on a record when interacting with an imported namespace.",
+        "The issue was caused because the import logic didn't define any model constraints."
+      ],
+      "how": [
+        "by making the import logic apply model constraints when defining the model, as well as defining the record revisions model."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/a554e289cb6087e47f6fd76e61dde2bd53312dc4"
+      ]
+    },
+    {
+      "what": [
+        "error when trying to fetch incorrectly configured workflow triggers in {APP_NAME_ADMIN}."
+      ],
+      "how": [
+        "by adding a check to determine if a constant has a name.",
+        "Also, the automatic addition of constraints after an event type was selected is removed."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/7a9802a1c74da0e58c6bf9e3b24767d80ab970e9"
+      ]
+    },
+    {
+      "what": [
+        "future/past values not being selectable if DateTime field has future/past only values option applied."
+      ],
+      "how": [
+        "by disabling the future/past only option on record list filter."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/24a31971e85cc279ee7eaddfda1e1d98b2580eaa",
+        "https://github.com/cortezaproject/corteza/commit/ffbaba27e8bc7f09f43117da55b8716693aa187b"
+      ]
+    },
+    {
+      "what": [
+        "{APP_NAME_ADMIN} user view not properly updated after 'Confirm email address' was clicked in {APP_NAME_ADMIN}."
+      ],
+      "how": [
+        "by refetching the user on {APP_NAME_ADMIN} user patch."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/e76fb09707239b8e8c7b430e1f829e4adb975894"
+      ]
+    },
+    {
+      "what": [
+        "field kind not updating after being selected."
+      ],
+      "how": [
+        "by updating the field kind after it is selected."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1018/commits/53418299193eb6a4b4610527c7b5ecb399efa85e",
+        "https://github.com/cortezaproject/corteza/pull/1018/commits/de26f561a7f6b24b79136c49a84971ab55b9dac5"
+      ]
+    },
+    {
+      "what": [
+        "record add or update error when dedicated record revisions table is configured."
+      ],
+      "how": [
+        "by rearranging the order of the rel_resource, revision, and operation columns when custom dedicated record revision table is configured and changed the operation column datatype to text."
```

Comment:
THe rearrange was done for consistency, it wasn't what caused the issue. The wrong datatype is what caused the issue.
Focus on the data type; you can mention the reorder if you want.